### ISSUE TITLE
Fuzzy search cannot index project

### ISSUE THREAD

**Issue by stdc105** [atom/fuzzy-finder#410]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

Unable to index the project when using ctrl-t or ctrl-p to open the file with fuzzy search after upgrading to 1.42.0.

### Steps to Reproduce

1. Open atom with a project.
2. Press ctrl-t or ctrl-p to open "Fuzzy Finder: Toggle File Finder"
3. Enter a filename that exist in current project

**Expected behavior:**
After indexing for a while, a list of files with matching filenames should appear and should be able to be selected.

**Actual behavior:**
Showing "Indexing project... 0" for a long time, and never finishing index.

**Reproduces how often:**
All the time. Constantly reproduced on Win10 1809, regardless of project size.

### Versions

Atom    : 1.42.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0

### Additional Information
Reproduced on Win10 1809. Tried with safe mode and after `atom --clear-window-state`.

**Comment by stdc105** [comment]
> I tried to reproducing on both `1.42.0-stable` and [`1.43.0-beta`](https://atom.io/beta) on both Mac and Windows and they are both working fine and rapidly, even on relatively large projects.
> 
> So we will need more details on the reproduction steps.
> 
> * Specially when it comes of how you open your project or is it a workspace?
> * You mentioned fuzzy search, did you mean fuzzy finder?
> * Can you clarify what did you mean by "open file" on the second step of reproduction?
> * Can you reproduce the bug when you use the command pallet `ctrl + shift + p` > type "fuzzy" > choose "Toggle File Finder" > type the [file name]
> * You could also try a classic system reboot for your PC that might resolve hidden issues related to memory, cache, buffer, etc.
> * If non of the above worked I would suggest taking a look at [this amazing article](https://www.lee-dohm.com/2015/01/04/writing-good-bug-reports/) by our community manager, that might help us reproduce this bug.
> 
> FYI, this issue belongs to https://github.com/atom/fuzzy-finder, so I am going to transfer the issue to the proper repo.

Thank you for your reply!
Sorry to use some inaccurate words when describing the issue. I'll take a look at the article and find out what other information I could help reproduce the bug.

I'll try to describe information I already have here, and update the issue description accordingly. 
- For opening a project, I tried using CLI `atom .` or open with right click menu on folder (Open with Atom) or open from an existing atom window with empty default editor.
- For fuzzy search, I mean fuzzy finder. Buffer finder and Git Status finder work, but File finder not (and this is the issue). The `open file` means File finder. Sorry again for using inaccurate words.
- I can reproduce the bug when using ctrl-shift-p with Toggle File Finder. Same behavior as using ctrl-b or ctrl-t (Showing "Indexing project... 0")
- I already tried reboot twice, and also already tried safe mode after each reboot. Same behavior.

I'll also try to reproduce the problem using another computer tonight.

**Comment by stdc105** [comment]
Sorry for the late update:
I have tried to reproduce the issue on my personal machine but it seems to be good. I have fully deleted all atom installation (including project cache) and reinstall atom, it seems the problem is solved. While upgrading from 1.42 to 1.43 there is no problem any more, so I am not really sure if there is anything wrong before I reinstalled all the data. Anyway, sorry for any inconveniences, and I'll close the issue for now.
Thank you for helping! At least I'll know how to better write a bug report next time.
